src/css/base.scss

```diff
@@ -856,6 +856,37 @@ button.anchor {
     background-color: #f5f5f5;
 }
 
+.override-button.cta {
```

Comment:
Is it used anywhere in the dashboard? if not can remove

Reply:
Yes it is used.